scripts/build_utils.sh

```diff
@@ -837,6 +837,12 @@ gen_debian_version() {
     echo "${raw}${bptag}"
 }
 
+maybe_set_jaeger() {
+    if (echo $BRANCH | grep -i "jaeger"); then
+      export WITH_JAEGER=true
+    fi
+}
```

Comment:
i think there is no point to enable jaeger support in 3 ways. one way would suffice.

Reply:
+1.  ceph-build.git is already a house of cards.  Let's not add more :smile:

Reply:
ack, got it, tried to keep it minimal this time